### ISSUE TITLE
Superhot Pushaway Power Controls

### ISSUE THREAD

**Issue by mfncreative** [LibreVR/Revive#425]
The grip buttons on the vive controller work fine, once you get to the force or push tutorial, none of the buttons seem to allow you to do that command.

**Comment by LibreVR** [comment]
Have you followed the Touch Basics tutorial in the Revive Dashboard to familiarize yourself with all the buttons that are provided by the Oculus Touch emulation?

**Comment by mfncreative** [comment]
I could be wrong here, I did that and in games like VR Sports Challenge, all buttons work.  Super Hot, only the grip buttons make the hands move.  I have only just started with revive on the current version.

**Comment by LibreVR** [comment]
I've never heard of this issue, can you find any youtube video demonstrating this part of Superhot VR?

**Comment by jaorg1234** [comment]
Are you talking about the "super power" you get later on in the game? As far as I remember you have to hold down the trigger buttons on both controllers to activate that ability. Afterwards you are only able to perform that action if your hands don't seem "glitchy".

**Comment by mfncreative** [comment]
Right, I can't get passed the part where it introduces it.   I'll try to test it again.

**Comment by mfncreative** [comment]
Restarted everything and it's working now.

**Comment by happytor1** [comment]
Hello, im having the same issue. Newest version of superhot. My vive controls for the grip button work fine until I get to the level where I unlock the "Super power". at this stage, I am unable to use the "super power" Restarting my game has yet to fix it.

**Comment by Armada651** [comment]
I finally got to this level and the super power works fine. But you have to press both the grip and the trigger on both controllers at the same time to use it.
